Gruntfile.js

```diff
@@ -388,7 +385,7 @@ module.exports = function(grunt) {
   var updateQuickTestsJSON = function() {
     var qtJSON = [];
     var rawtests = grunt.file.expand("quicktests/overlaying/tests/**/*.js");
-    rawtests.forEach(function(value, index, array){
+    rawtests.forEach(function(value){
```

Comment:
`"args": "after-used"` means that if `array` is used in the above, then the rule ignores the `value` and the `index` parameter usage.  However, in the example above, `index` and `array` are just not used.

I personally prefer the slim approach, only exposing what one needs, but I can see the validity in your statement.  We can do a quick survey to see what we prefer